### PR TITLE
move vaults to inactive

### PR DESCRIPTION
**Pull request by Crypto-One-dev**

### CODE REVIEW

File: data/mainnet/tokens.js [hunk 1]
```diff
@@ -706,11 +706,12 @@ module.exports = {
     },
   },
   quickswap_PSP_MATIC: {
+    inactive: true,
     chain: CHAINS_ID.MATIC_MAINNET,
-    category: VAULT_CATEGORIES_IDS.QUICKSWAP,
+    category: VAULT_CATEGORIES_IDS.INACTIVE,
```
Comment (hubert895) [inline data/mainnet/tokens.js:711]:
```suggestion
    category: VAULT_CATEGORIES_IDS.INACTIVE_POLYGON,
```

File: data/mainnet/tokens.js [hunk 2]
```diff
@@ -757,12 +758,13 @@ module.exports = {
     cmcRewardTokenSymbols: ['miFARM', 'pUSDC', 'pWETH'],
   },
   popsicle_ICE_WETH: {
+    inactive: true,
     chain: CHAINS_ID.MATIC_MAINNET,
-    category: VAULT_CATEGORIES_IDS.POPSICLE,
+    category: VAULT_CATEGORIES_IDS.INACTIVE,
```
Comment (hubert895) [inline data/mainnet/tokens.js:763]:
```suggestion
    category: VAULT_CATEGORIES_IDS.INACTIVE_POLYGON,
```

File: data/mainnet/tokens.js [hunk 3]
```diff
@@ -4027,11 +4031,12 @@ module.exports = {
     ],
   },
   pancake_CAKE: {
+    inactive: true,
     chain: CHAINS_ID.BSC_MAINNET,
-    category: VAULT_CATEGORIES_IDS.PANCAKE,
+    category: VAULT_CATEGORIES_IDS.INACTIVE,
```
Comment (hubert895) [inline data/mainnet/tokens.js:4036]:
```suggestion
    category: VAULT_CATEGORIES_IDS.INACTIVE_BSC,
```

File: data/mainnet/tokens.js [hunk 4]
```diff
@@ -4067,11 +4072,12 @@ module.exports = {
     ],
   },
   pancake_USDT_BNB: {
+    inactive: true,
     chain: CHAINS_ID.BSC_MAINNET,
-    category: VAULT_CATEGORIES_IDS.PANCAKE,
+    category: VAULT_CATEGORIES_IDS.INACTIVE,
```
Comment (hubert895) [inline data/mainnet/tokens.js:4077]:
```suggestion
    category: VAULT_CATEGORIES_IDS.INACTIVE_BSC,
```

File: data/mainnet/tokens.js [hunk 5]
```diff
@@ -4374,11 +4380,12 @@ module.exports = {
     },
   },
   goose_EGG_BNB: {
+    inactive: true,
     chain: CHAINS_ID.BSC_MAINNET,
-    category: VAULT_CATEGORIES_IDS.GOOSE,
+    category: VAULT_CATEGORIES_IDS.INACTIVE,
```
Comment (hubert895) [inline data/mainnet/tokens.js:4385]:
```suggestion
    category: VAULT_CATEGORIES_IDS.INACTIVE_BSC,
```

File: data/mainnet/tokens.js [hunk 6]
```diff
@@ -4395,11 +4402,12 @@ module.exports = {
     ],
   },
   goose_EGG_BUSD: {
+    inactive: true,
     chain: CHAINS_ID.BSC_MAINNET,
-    category: VAULT_CATEGORIES_IDS.GOOSE,
+    category: VAULT_CATEGORIES_IDS.INACTIVE,
```
Comment (hubert895) [inline data/mainnet/tokens.js:4407]:
```suggestion
    category: VAULT_CATEGORIES_IDS.INACTIVE_BSC,
```

File: data/mainnet/tokens.js [hunk 7]
```diff
@@ -4818,11 +4826,12 @@ module.exports = {
     ],
   },
   ellipsis_EPS_BNB: {
+    inactive: true,
     chain: CHAINS_ID.BSC_MAINNET,
-    category: VAULT_CATEGORIES_IDS.ELLIPSIS,
+    category: VAULT_CATEGORIES_IDS.INACTIVE,
```
Comment (hubert895) [inline data/mainnet/tokens.js:4831]:
```suggestion
    category: VAULT_CATEGORIES_IDS.INACTIVE_BSC,
```

File: data/mainnet/tokens.js [hunk 8]
```diff
@@ -4915,11 +4924,12 @@ module.exports = {
     ],
   },
   ellipsis_BTC: {
+    inactive: true,
     chain: CHAINS_ID.BSC_MAINNET,
-    category: VAULT_CATEGORIES_IDS.ELLIPSIS,
+    category: VAULT_CATEGORIES_IDS.INACTIVE,
```
Comment (hubert895) [inline data/mainnet/tokens.js:4929]:
```suggestion
    category: VAULT_CATEGORIES_IDS.INACTIVE_BSC,
```

File: data/mainnet/tokens.js [hunk 9]
```diff
@@ -6285,11 +6295,12 @@ module.exports = {
     cmcRewardTokenSymbols: ['iFARM', 'DEN'],
   },
   jarvis_JEUR_USDC_HODL: {
-    category: VAULT_CATEGORIES_IDS.JARVIS,
+    inactive: true,
+    category: VAULT_CATEGORIES_IDS.INACTIVE,
```
Comment (hubert895) [inline data/mainnet/tokens.js:6299]:
```suggestion
    category: VAULT_CATEGORIES_IDS.INACTIVE_POLYGON,
```

File: data/mainnet/tokens.js [hunk 10]
```diff
@@ -6318,11 +6329,12 @@ module.exports = {
     cmcRewardTokenSymbols: ['iFARM', 'AUR-APR22'],
   },
   jarvis_JGBP_USDC_HODL: {
-    category: VAULT_CATEGORIES_IDS.JARVIS,
+    inactive: true,
+    category: VAULT_CATEGORIES_IDS.INACTIVE,
```
Comment (hubert895) [inline data/mainnet/tokens.js:6333]:
```suggestion
    category: VAULT_CATEGORIES_IDS.INACTIVE_POLYGON,
```

File: data/mainnet/tokens.js [hunk 11]
```diff
@@ -6351,11 +6363,12 @@ module.exports = {
     cmcRewardTokenSymbols: ['iFARM', 'AUR-APR22'],
   },
   jarvis_JCHF_USDC_HODL: {
-    category: VAULT_CATEGORIES_IDS.JARVIS,
+    inactive: true,
+    category: VAULT_CATEGORIES_IDS.INACTIVE,
```
Comment (hubert895) [inline data/mainnet/tokens.js:6367]:
```suggestion
    category: VAULT_CATEGORIES_IDS.INACTIVE_POLYGON,
```

File: data/mainnet/tokens.js [hunk 12]
```diff
@@ -6384,11 +6397,12 @@ module.exports = {
     cmcRewardTokenSymbols: ['iFARM', 'AUR-APR22'],
   },
   jarvis_AUR3_USDC: {
-    category: VAULT_CATEGORIES_IDS.JARVIS,
+    inactive: true,
+    category: VAULT_CATEGORIES_IDS.INACTIVE,
```
Comment (hubert895) [inline data/mainnet/tokens.js:6401]:
```suggestion
    category: VAULT_CATEGORIES_IDS.INACTIVE_POLYGON,
```

**Review by hubert895 — CHANGES_REQUESTED** [review]
Incorrect categories

**Comment by CryptJS13** [comment]
Please also change the hardwork settings in the `addresses.json` file

**Review by hubert895 — APPROVED** [review]

**Comment by Crypto-One-dev** [comment]
> Please also change the hardwork settings in the `addresses.json` file

Fixed for dohardwork settings. Please check

**Pull request merged**